src/test/java/de/mediathekview/mserver/crawler/swr/parser/SwrFilmDeserializerTest.java

```diff
@@ -0,0 +1,98 @@
+package de.mediathekview.mserver.crawler.swr.parser;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.google.gson.JsonElement;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.testhelper.AssertFilm;
+import de.mediathekview.mserver.testhelper.JsonFileReader;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class SwrFilmDeserializerTest {
+
+  @Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
+        {
+            "/swr/swr_film_detail.json",
+            "Landesschau Baden-Württemberg",
+            "Landesschau Baden-Württemberg vom 12.09.2018",
+            LocalDateTime.of(2018, 9, 12, 18, 45, 0),
+            Duration.ofMinutes(44).plusSeconds(49),
+            "Mit diesen Themen unter anderem: Anti-Terror-Übung am Stuttgarter Hauptbahnhof *** Neue Medikament zur Behandlung von Migräne zugelassen *** Zu Gast ist Lars Eidinger, Schauspieler, der die Rolle des Bertolt Brecht übernommen hat.",
+            "https://swrmediathek.de/player.htm?show=bcdd80a0-b6bc-11e8-b070-005056a12b4c",
+            "https://pdodswr-a.akamaihd.net/swr/swr-fernsehen/landesschau-bw/00-hauptbeitrag/1053798.m.mp4",
+            "https://pdodswr-a.akamaihd.net/swr/swr-fernsehen/landesschau-bw/00-hauptbeitrag/1053798.l.mp4",
+            "https://pdodswr-a.akamaihd.net/swr/swr-fernsehen/landesschau-bw/00-hauptbeitrag/1053798.xl.mp4",
+            "https://subtitles.swr.de/swr/swr-fernsehen/landesschau-bw/00-hauptbeitrag/1053798.xml"
+        }
+    });
+  }
+
+  private final String jsonFile;
+  private final String theme;
+  private final String title;
+  private final LocalDateTime dateTime;
+  private final Duration duration;
+  private final String description;
+  private final String website;
+  private final String smallUrl;
+  private final String normalUrl;
+  private final String hdUrl;
+  private final String subtitleUrl;
+
+  public SwrFilmDeserializerTest(String aJsonFile, String aTheme, String aTitle,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 11 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)